src/makeCalculator.js

```diff
@@ -37,7 +37,51 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      calculator.result = 0;
```

Comment:
right now we can execute this code:
![image](https://user-images.githubusercontent.com/37161315/197532458-75fa2b43-2863-46af-8f78-1164abd7163e.png)
i guess it should work a bit other way:
- we should change object that we called `operate` **on**,
- operation shouldn't be dependant on exact object that you call it from
- you can somehow pass needed object from operate to this actual `operation`, try `this`